evaluation/evaluate_text_generation.py

```diff
@@ -1,14 +1,18 @@
 import numpy as np
 from datasets import load_dataset
-from sacrebleu import corpus_bleu
 from transformers import pipeline
-
+from rouge_score import rouge_scorer
 from dataset import KeyValueDataset
 from tasks.TaskTypes import TaskType
 
+scorer = rouge_scorer.RougeScorer(['rouge1'])
+
 
-def sacrebleu_score(hypotheses, references):
-    return corpus_bleu(hypotheses, [references]).score
+def rogue_score(hypotheses, references):
+    total_f1_scores = 0.0
+    for ref, hyp in zip(references, hypotheses):
+        total_f1_scores+=scorer.score(ref, hyp)['rouge1'][2] # [2] for f1 score
```

Comment:
Why ROUGE-1?

Reply:
rouge1 is matching unigrams between hypothesis and reference. We can also use rouge2(bigram) along with rouge1 if needed.

Reply:
Hi Sebastian, I have also added "ROUGE-2" and "ROUGE-L" in the evaluation. Please have a look.

Reply:
Thanks